utils/build-script-impl

```diff
@@ -3881,6 +3881,8 @@ for host in "${ALL_HOSTS[@]}"; do
             (cd "${INSTALL_SYMROOT}" &&
              find ./"${CURRENT_PREFIX}" -perm -0111 -type f -print | \
                grep -v crashlog.py | \
+               grep -v swift-api-dump.py | \
+               grep -v swift_build_sdk_interfaces.py | \
                grep -v symbolication.py | \
```

Comment:
Should we just `grep -v .py` and be done with it?

Reply:
I can make that change if you like.